owrx/reporting/mqtt.py

```diff
@@ -40,7 +40,11 @@ def _getClient(self):
         host = parts[0]
         if len(parts) > 1:
             port = int(parts[1])
-        client.connect(host=host, port=port)
+
+        try:
+            client.connect(host=host, port=port)
+        except Exception as e:
+            logger.debug("Exception connecting: " + str(e))
```

Comment:
I'd prefer to pass this to `logger.exception` for two reasons: First of all, it will show up in the logs even when debugging is off (which is probably the only way for the user to figure out the config may be erroneous), and second because that will also include the call stack.

Reply:
The call stack is going to be sort of useless in this case, since we already know exactly where it crashed, but anyway, changed to logger.exception()